docs/variants/pc_engines/releases_uefi.md

```diff
@@ -95,7 +95,7 @@ using [this key](https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/da
 - [AMD System Management Unit - Software Configuration Settings binary based on 1433 revision a8db7dfe](https://github.com/coreboot/blobs/tree/a8db7dfe/southbridge/amd/avalon/PSP/SmuScs.bin)
 - [AMD Hudson xHCI firmware based on 1.1.0.0068 revision a8db7dfe](https://github.com/coreboot/blobs/tree/a8db7dfe/southbridge/amd/avalon/xhci.bin)
 
-[newsletter]: https://newsletter.3mdeb.com/subscription/78gF-_gm1
+[newsletter]: https://newsletter.3mdeb.com/subscription/ReBpt3IZY
```

Comment:
What about subscribers of previous newsletter `78gF-_gm1`? Any justification of this change?

BTW `ReBpt3IZY` doesn't exist.

Reply:
@miczyg1 `78gF-_gm1` is [Dasharo (coreboot+UEFI) Entry Subscription for Network Appliance Newsletter](https://newsletter.3mdeb.com/lists/view/80) since this was copypasta from Dasharo (coreboot+UEFI) v0.9.0

> BTW ReBpt3IZY doesn't exist.

?

![newsletter_exist](https://github.com/Dasharo/docs/assets/1071894/93b6a736-2434-4b64-a7b3-eda4842b3b5f)

Reply:
Feel free to reopen this conversation if it is still an issue. I have to close it now to meet all merging criteria.

Reply:
Weird. I tried to find it on the mailtrain instance, but the search engine gave me no results... Now I see it is there because it was 51st position on the list (initially only 50 were displayed). My bad.

@pietrushnic however my point is why do you modify the newsletter for UEFI releases?!

Reply:
@miczyg1 this is a very valid point. I'm doing it for two reasons:
- Because linking to the newsletter, which says, `The list does not allow public subscriptions,` is not helpful.
- Because one could be interested in getting notifications but not interested in buying a subscription, or maybe someone could consider it depending on the frequency of releases, we need a channel for those and similar cases.

Overall, giving people that option for all releases is way more helpful from the business perspective than a link to a list to which you cannot subscribe.

Reply:
Allright...

So it seems to go into a direction of having multiple newsletters for one firmware flavor. I wonder who will keep up with this and pay attention which lists should be used and when, under what conditions. Very automation-friendly. Just saying...

Reply:
@miczyg1 business first, if you don't have revenue, you don't have any automation. There is no value in using a mailing list to which no one can subscribe. By the way, it was an idea from the beginning, but probably not explained correctly by me, that we want to have a notification mailing list for non-subscribers.

I am still determining how having a link to a private list helps with automation. Mailing list management is something we have to improve. Still, I'm unsure if we have enough experience or time to generate a process that could stabilize the naming convention for lists and relation to all (changing) flavors of Dasharo. Keep calm, and let's improve Dasharo every time we release.

Reply:
> I am still determining how having a link to a private list helps with automation

What I meant is that we will have to add more complexity to the release and newsletter automation that we currently have.